specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/examples/PrivateEndpointConnections_Create_MaximumSet_Gen.json

```diff
@@ -0,0 +1,84 @@
+{
+  "title": "PrivateEndpointConnections_Create",
+  "operationId": "PrivateEndpointConnections_Create",
+  "parameters": {
+    "subscriptionId": "48841AC8-BAB7-41CD-A060-F539444ACD50",
+    "resourceGroupName": "rgelasticsan",
+    "elasticSanName": "1jRHU_51x_8f_142",
+    "privateEndpointConnectionName": "djwitfgttbuofvxnju",
+    "parameters": {
+      "properties": {
+        "privateEndpoint": {},
+        "privateLinkServiceConnectionState": {
+          "status": "Pending",
+          "description": "ztdqqordnivv",
+          "actionsRequired": "tdddlgyurtpciltyenvhftkhrvyure"
+        },
+        "groupIds": [
+          {}
+        ]
+      },
+      "tags": {
+        "key4273": "lmmhzxoliqxko"
+      }
+    },
+    "api-version": "2022-12-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "provisioningState": "Invalid",
+          "privateEndpoint": {
+            "id": "jwdiwapuudxmoswvanzxzxgmvcak"
+          },
+          "privateLinkServiceConnectionState": {
+            "status": "Pending",
+            "description": "ztdqqordnivv",
+            "actionsRequired": "tdddlgyurtpciltyenvhftkhrvyure"
+          },
+          "groupIds": [
+            {
+              "id": "jwdiwapuudxmoswvanzxzxgmvcak"
+            }
+          ]
+        },
+        "id": "nrkgifbkkznauosebqbwvtlqzuj",
+        "name": "ih",
+        "type": "thqoluwytlweobwrbbfx",
```

Comment:
Could you give a more meaningful value for the type?

Reply:
I thought description can help for rest API doc, this types is a string object and values are auto-generated.
It would be very tough for generating manual examples.

Reply:
As this will in the rest API doc, and normally the value should not be a random string, please give a meaningful value.
We don't want the rest API doc look not so professional.

Reply:
Here's a sample for your reference.
https://github.com/Azure/azure-rest-api-specs/blob/02acc29a5618a6998469a7fcc750e301f130a19f/specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/examples/StorageAccountGetPrivateEndpointConnection.json#L15

Please try to avoid using random string in all sample field, and use meaningful string with correct format, to better guild customer to use our rest API.